.travis.yml

```diff
@@ -136,25 +146,31 @@ before_install:
     fi
   - python -V -V
   - python -c 'import sys; print("Python debug build:", hasattr(sys, "gettotalrefcount"))'
-  - travis_retry pip install --upgrade pip setuptools wheel
-  - travis_retry pip install cython
-  - if [ -n "$NUMPYSPEC" ]; then travis_retry pip install $NUMPYSPEC; fi
-  - travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
+  - |
+    if [ "${TRAVIS_CPU_ARCH}" != "arm64" ]; then
+        travis_retry pip install --upgrade pip setuptools wheel
+        travis_retry pip install cython
+    fi
+  - |
+    if [[ "$NUMPYSPEC" && "$TRAVIS_CPU_ARCH" != "arm64" ]]; then travis_retry pip install $NUMPYSPEC; fi
+  - |
+    if [ "${TRAVIS_CPU_ARCH}" != "arm64" ]; then
+        travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
+    fi
   - |
     if [ -n "${USE_DEBUG}" ]; then
         # see gh-10676; need to pin pytest version with debug
         # Python 3.6
         travis_retry pip install pytest==5.0.1
     fi
-  - travis_retry pip install gmpy2  # speeds up mpmath (scipy.special tests)
-  - travis_retry pip install pybind11
+  - if [[ "${TRAVIS_CPU_ARCH}" != "arm64" ]]; then travis_retry pip install gmpy2 pybind11; fi
   - |
     if [ -z "${USE_DEBUG}" -a "${TRAVIS_CPU_ARCH}" == "amd64" -a "${TRAVIS_PYTHON_VERSION}" != "3.8" ]; then
         travis_retry pip install numba==0.47
         travis_retry pip install --only-binary=:all: sparse
     fi
   - |
-    if [ "${TESTMODE}" == "full" ]; then
+    if [ "${TESTMODE}" == "full" -a "${TRAVIS_CPU_ARCH}" != "arm64" ]; then
```

Comment:
This change can be undone, it's not needed because there's no arm64 test with "full".